lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.ts

```diff
@@ -190,17 +190,17 @@ export class PeopleCloudComponent implements OnInit, OnChanges {
                 this.logService.error(error);
             }
             const isUserValid: Boolean = this.userExists(result);
-            return isUserValid ? new IdentityUserModel(result[0]) : user;
+            return isUserValid ? new IdentityUserModel(result) : user;
         });
         return await Promise.all(promiseBatch);
     }
 
     async searchUser(user: IdentityUserModel) {
         const key: string = Object.keys(user)[0];
         switch (key) {
-            case 'id': return this.identityUserService.findUserById(user[key]).toPromise();
-            case 'username': return this.identityUserService.findUserByUsername(user[key]).toPromise();
-            case 'email': return this.identityUserService.findUserByEmail(user[key]).toPromise();
+            case 'id': return await this.identityUserService.findUserById(user[key]).toPromise();
+            case 'username': return (await this.identityUserService.findUserByUsername(user[key]).toPromise())[0];
```

Comment:
(await this.identityUserService.findUserByUsername(user[key]).toPromise())[0]; it get the first of the array right? what happens if the array is empty?